src/v1/sources/shopify/transform.js

```diff
@@ -15,6 +36,8 @@ const process = async (inputEvent) => {
   }
   // this is for common logic for server-side events processing for both pixel and tracker apps.
   const response = await processWebhookEvents(event);
```

Comment:
On the other note, `eventOrigin: server` is missing for server-side events - https://github.com/rudderlabs/rudder-transformer/blob/develop/src/v0/sources/shopify/transform.js#L198-L199
This change would happen for existing tracker app based too - have quick impact check before doing the change or handle in different PR

Reply:
I think we had decided against putting this, as we only wanted a sort of differentiator or label for pixel events, and since events from webhooks are exactly same for both app flows.
LMK if we do want to keep it still.

Reply:
we discussed of both client/server values for eventOrigin during finalisation of this variable - can you help me in reminding what was the reason for not adding it for server-side events ?